### PR TITLE
IRIS-4268 | Increased the line-height and align text right

### PR DESCRIPTION
**Pull request by slayful**
## Links

* http://wikia-inc.atlassian.net/browse/IRIS-4268

### CODE REVIEW

File: front/main/app/styles/component/_wikia-user-profile.scss [hunk 1]
```diff
@@ -66,17 +66,18 @@
 		padding: 0 0 11px;
 
 		.wds-notifications__mark-all-as-read {
-			line-height: 12px;
+			line-height: 14px;
```
Comment (kvas-damian) [inline front/main/app/styles/component/_wikia-user-profile.scss]: Why don't we use variables from DS?

File: front/main/app/mixins/notifications-unread-count.js [hunk 2]
```diff
@@ -3,6 +3,12 @@ import Ember from 'ember';
...
-	unreadCount: computed.oneWay('notifications.model.unreadCount'),
-	hasUnread: computed.gt('notifications.model.unreadCount', 0),
+	unreadCount: computed('notifications.model.unreadCount', function() {
+		const count = this.get('notifications.model.unreadCount');
+		if (count > 99) {
+			return '99+';
+		} else {
+			return count;
+		}
+	}),	hasUnread: computed.gt('notifications.model.unreadCount', 0),
```
Comment (kvas-damian) [inline front/main/app/mixins/notifications-unread-count.js]: missing `nl`

File: front/main/app/styles/component/_wikia-user-profile.scss (hunk 1 above)
Comment (slayful) [inline front/main/app/styles/component/_wikia-user-profile.scss]: In DS there's only `$wds-line-height: 1.5em;`, which's not that.
I could use `$wds-typescale-size-` but I'm not sure this is appropriate.

Comment (kvas-damian) [inline front/main/app/styles/component/_wikia-user-profile.scss]: In this case I think you should contact Designer and advice them to use DS defined sizes.

**Pull request merged**